File: Atividades/atv3/04_Operações básicas/Euler1.js
module.exports.euler1 = (n)=>{
    let xs = []
    for(let i=1;i<n;i++) 
        xs.push(i) 

    let aux = xs.filter(x=> x%3==0 || x%5==0)
    return aux.reduce((ac,x)=>ac+x)

}

File: Atividades/atv03/04_Operações básicas/Euler1.hs
module Euler1 (euler1) where

--Calcule a soma de todos os números de 1 até n que são múltiplos de 3 ou 5. Utilize uma única list compreension.
euler1 :: Int -> Int
euler1 n = sum (filter (\x -> (x `mod` 3)==0 ||(x `mod` 5)==0 ) [1..(n-1)] )  
